tracer/src/Datadog.Trace/Generated/netstandard2.0/Datadog.Trace.SourceGenerators/AspectsDefinitionsGenerator/AspectsDefinitions.g.cs

```diff
@@ -17,477 +17,477 @@ internal static partial class AspectDefinitions
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadData(System.String,System.String,System.Byte[])\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadDataTaskAsync(System.String,System.Byte[])\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadDataTaskAsync(System.String,System.String,System.Byte[])\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadFile(System.String,System.String)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadFile(System.String,System.String,System.String)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadFileTaskAsync(System.String,System.String)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadFileTaskAsync(System.String,System.String,System.String)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadString(System.String,System.String)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadString(System.String,System.String,System.String)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadStringTaskAsync(System.String,System.String)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadStringTaskAsync(System.String,System.String,System.String)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadValues(System.String,System.Collections.Specialized.NameValueCollection)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadValues(System.String,System.String,System.Collections.Specialized.NameValueCollection)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadValuesTaskAsync(System.String,System.Collections.Specialized.NameValueCollection)\",\"\",[1],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::UploadValuesTaskAsync(System.String,System.String,System.Collections.Specialized.NameValueCollection)\",\"\",[2],[False],[None],Propagation,[])] Review(System.String)",
-"  [AspectMethodInsertBefore(\"System.Net.WebClient::set_BaseAddress(System.String)\",\"\",[0],[False],[None],Propagation,[])] Review(System.String)",
```

Comment:
I assume all the `Propagation` -> `Debug` is intentional, right?

Reply:
Yes, it's `Default`. It was a pending bug we had to fix (it is usage metadata)